FSIM_USAGE.md

```diff
@@ -0,0 +1,170 @@
+```bash
+go-fdo-server owner 127.0.0.1:8043 \
+  --db /tmp/fdo/db/own.db --db-pass "$DB_PASS" \
+  --device-ca-cert /tmp/fdo/keys/device_ca_cert.pem \
+  --owner-key /tmp/fdo/keys/owner_key.der \
+  --command-date
+```
+
+### Available Commands
+- `--command-date`: Executes `date --utc` on the device to display current UTC time
+
+### Example Output
+When a device onboards with the command FSIM enabled, the device will execute the specified command and the output will be displayed in the owner service logs.
+
+## fdo.download FSIM
+
+### Purpose
+Transfer files from the owner server to the device during onboarding.
+
+### Usage
+```bash
+go-fdo-server owner 127.0.0.1:8043 \
+  --db /tmp/fdo/db/own.db --db-pass "$DB_PASS" \
+  --device-ca-cert /tmp/fdo/keys/device_ca_cert.pem \
+  --owner-key /tmp/fdo/keys/owner_key.der \
+  --command-download /path/to/local/file1.txt \
+  --command-download /path/to/local/file2.conf
+```
+
+### Parameters
+- `--command-download <file_path>`: Specify a local file path to transfer to the device
+- Flag can be used multiple times to transfer multiple files
+
+### Example
+```bash
+# Prepare files to download
+echo "Device configuration data" > /tmp/device-config.txt
+echo "Application settings" > /tmp/app-settings.json
+
+# Start owner with download FSIM
+go-fdo-server owner 127.0.0.1:8043 \
+  --db /tmp/fdo/db/own.db --db-pass "$DB_PASS" \
+  --device-ca-cert /tmp/fdo/keys/device_ca_cert.pem \
+  --owner-key /tmp/fdo/keys/owner_key.der \
+  --command-download /tmp/device-config.txt \
+  --command-download /tmp/app-settings.json
+```
+
+### Error Handling
+- If a specified file cannot be opened, the owner service will log a fatal error
+- File permissions on the owner server must allow reading by the service process
+
+## fdo.upload FSIM
+
+### Purpose
+Transfer files from the device to the owner server during onboarding.
+
+### Usage
+```bash
+go-fdo-server owner 127.0.0.1:8043 \
+  --db /tmp/fdo/db/own.db --db-pass "$DB_PASS" \
+  --device-ca-cert /tmp/fdo/keys/device_ca_cert.pem \
+  --owner-key /tmp/fdo/keys/owner_key.der \
+  --upload-directory /tmp/uploads \
+  --command-upload device-logs.txt \
+  --command-upload system-info.json
+```
+
+### Parameters
+- `--upload-directory <dir_path>`: Directory on owner server where uploaded files will be stored
+- `--command-upload <filename>`: Name of file to request from device
+- Upload flag can be used multiple times for multiple files
+
+### Example
+```bash
+# Create upload directory
+mkdir -p /tmp/fdo-uploads
+
+# Start owner with upload FSIM
+go-fdo-server owner 127.0.0.1:8043 \
+  --db /tmp/fdo/db/own.db --db-pass "$DB_PASS" \
+  --device-ca-cert /tmp/fdo/keys/device_ca_cert.pem \
+  --owner-key /tmp/fdo/keys/owner_key.der \
+  --upload-directory /tmp/fdo-uploads \
+  --command-upload /var/log/device.log \
+  --command-upload /etc/device-id.txt
+```
+
+## fdo.wget FSIM
+
+### Purpose
+Instruct the device to download files from external URLs during onboarding.
```

Comment:
Probably should add some text explaining that the device will open a new TCP connection to the HTTP server over which it will download the files rather than using the encrypted connection established between device/owner by FIDO.

The important point is that the user needs to understand the security ramifications involved.

And the go-fdo-client needs to have the --wget-dir <dir> option provided in order for this all to work. <dir> is the path to a directory on the device where the files will be downloaded to.  They retain the filenames given in the --command-wget argument.